sdk/testnet_consts.go

```diff
@@ -39,6 +39,7 @@ var knownTestnetTokenbridgeEmitters = map[vaa.ChainID]string{
 	vaa.ChainIDXLayer:          "000000000000000000000000dA91a06299BBF302091B053c6B9EF86Eff0f930D",
 	vaa.ChainIDLinea:           "000000000000000000000000C7A204bDBFe983FCD8d8E61D02b475D4073fF97e",
 	vaa.ChainIDBerachain:       "000000000000000000000000a10f2eF61dE1f19f586ab8B6F2EbA89bACE63F7a",
+	vaa.ChainIDSnaxchain:       "000000000000000000000000a10f2eF61dE1f19f586ab8B6F2EbA89bACE63F7a",
```

Comment:
Should this have the same value as Berchain?

Reply:
Yup. Some times the contracts get deployed to the same address on different chains. If you see the link for the token bridge up in the PR description, you'll see that's the token bridge.

Reply:
Ah, thanks. I saw that the contract address matched what you posted but I just wanted to double-check that a duplicate value is safe. 👍🏻